app/models/lti_context.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+# == Schema Information
+#
+# Table name: lti_contexts
+#
+#  id                 :integer          not null, primary key
+#  user_id            :integer          not null, foreign_key
+#  user_lti_id        :string(255)      not null
+#  context_id         :string(255)      not null
+#  lms_id             :string(255)      not null
+#  lms_family         :string(255)
+#
+
```

Comment:
Comments defining the last four attributes here would be helpful, including how they translate to fields in a Canvas instance.